lib/presentation/home/widget/carouselItemWidget.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:ttn_flix/presentation/themes/ttn_flix_spacing.dart';
+import 'package:ttn_flix/utils/ttn_flix_constant.dart';
+import '../../themes/ttn_flix_text_style.dart';
+
+class CarouselItemWidget extends StatelessWidget {
+  final String item;
+
+  const CarouselItemWidget({super.key, required this.item});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Container(
+          height: 225,
```

Comment:
@AM5324 instead of hardcoding height or width you can use media query to get the height or width of the screen and divide that with some aspect ration to get the dynamic height and width.